nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/processors/script/InvokeScriptedProcessor.java

```diff
@@ -92,11 +92,10 @@
                     logger.error(message, t);
                 }
             }
-        } else {
-            // Return defaults for now
-            relationships.add(REL_SUCCESS);
-            relationships.add(REL_FAILURE);
         }
+        // Add defaults
+        relationships.add(REL_SUCCESS);
+        relationships.add(REL_FAILURE);
```

Comment:
Yeah if this is failing tests, I'd say take a look at the tests. We don't want to add default relationships, especially when there's a legit instance of a Processor, as it is the Processor's responsibility to define all relationships